### PR TITLE
i18n(subscriber): replace subscriber hard-coded strings for env.t

### PR DESCRIPTION
**Pull request by marcelosizer**
With [habitrpg-shared #121](https://github.com/HabitRPG/habitrpg-shared/pull/121). Double check this one please.

(cc @paglias)

### CODE REVIEW

**Comment by lefnire** [comment]
I'm thinking we should pull everything out of lefnire/habitrpg-private#premium into the main repo. It might cause conflict with this PR, so we may need to do it concurrently/first. My thought is that branch doesn't provide any biz logic, just ccard processing and client stuff. @paglias @colegleason any objections? If not, I'll pull it in soon and just leave the bower reference on the private branch, and leave habitrpg-shared to handle the private stuff

**Comment by colegleason** [comment]
No objections from me.

On Tue, Jan 28, 2014 at 6:42 PM, Tyler Renelle notifications@github.comwrote:

> I'm thinking we should pull everything out of
> lefnire/habitrpg-private#premium into the main repo. It might cause
> conflict with this PR, so we may need to do it concurrently/first. My
> thought is that branch doesn't provide any biz logic, just ccard processing
> and client stuff. @paglias https://github.com/paglias @colegleasonhttps://github.com/colegleasonany objections? If not, I'll pull it in soon and just leave the bower
> reference on the private branch, and leave habitrpg-shared to handle the
> private stuff
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HabitRPG/habitrpg/pull/2627#issuecomment-33545283
> .

## 

## Cole Gleason

Student, University of Illinois at Urbana-Champaign
Email: cg@colegleason.com
Website: colegleason.com

**Comment by marcelosizer** [comment]
@lefnire let me know when it's done so I can edit the rest.

**Comment by paglias** [comment]
@lefnire for me too it's ok, can we merge this one or we shall wait? If you think it'll create conflicts create a PR and I'll merge it manyally

**Comment by lefnire** [comment]
@Sinza- @paglias ignore me, turns out I had already merged all the UI stuff for premium into develop. It's just server stuff that's unmerged, I'll deal with that later

File: views/options/settings.jade [hunk 1]
```diff
@@ -147,16 +145,100 @@ script(id='partials/feature-matrix-check.html',type='text/ng-template')
...
-          i.icon-ok
-          | Subscribed
+          i.icon-ok=env.t('subscribed')
         div(ng-include="'partials/options.settings.subscription.perks.html'")
-      .btn.btn-small.btn-danger(ng-click='cancelSubscription()') Cancel Subscription
\ No newline at end of file
+      .btn.btn-small.btn-danger(ng-click='cancelSubscription()')=env.t('cancelSub')
+
+
+  //-.well.subscription-features
```
Comment (paglias) [inline views/options/settings.jade:162]: I'm merging, but this part below where it comes from?

Comment (lefnire) [inline views/options/settings.jade:162]: Looks like it's using old code for the subscriptions details. Can you check to make sure this commit didn't clobber https://github.com/HabitRPG/habitrpg/commit/66f95cdd4cfb698fddc765a77b66d29e31eb1361 ?

Comment (paglias) [inline views/options/settings.jade:162]: It seems to have been copied from the old plans, but this is the settings page.... Removed in https://github.com/HabitRPG/habitrpg/commit/5f0cf6575c0dc4cfc041956e3dc27898d8b4242d

**Comment by lefnire** [comment]
Ruh-roh: http://gyazo.com/5934aa2d2e8063761eef4154f372d64d

**Comment by marcelosizer** [comment]
Let me see that, I'm finishing up the rest of the string interpolation, almost ready to go.

**Comment by marcelosizer** [comment]
@lefnire check the habitrpg-shared to see if the app.json has the subscriber.json line and if the subscriber.json file is there.

**Comment by marcelosizer** [comment]
@lefnire please see #2661 and [habitrpg-shared #122](https://github.com/HabitRPG/habitrpg-shared/pull/122) to see if it's fixed!

**Comment by lefnire** [comment]
you rock! :+1:

**Pull request merged**